test/e2e/specs/site-editor/homepage-settings.spec.js

```diff
@@ -0,0 +1,95 @@
+/**
```

Comment:
Thanks for providing an end to end test, this is a bit of a complex action and this provides some peace of mind.